### ISSUE TITLE
void-function bind-map-set-keys

### ISSUE THREAD

**Issue by shackra** [abingham/emacs-traad#28]
Found this bug on `GNU Emacs 26.1 (build 1, x86_64-w64-mingw32) of 2018-05-30`

Disabling the package seems like a good workaround.

```
Debugger entered--Lisp error: (void-function bind-map-set-keys)
  bind-map-set-keys((keymap keymap (header-line keymap (mouse-2 . push-button)) (mode-line keymap (mouse-2 . push-button)) (mouse-2 . push-button) (13 . push-button)) "RET" traad--button-action-visit-location "TAB" traad--button-action-preview-location)
  byte-code("\302\010\011\"\210\303\010\304\305\306\307%\207" [traad--location-button-keymap button-map set-keymap-parent bind-map-set-keys "RET" traad--button-action-visit-location "TAB" traad--button-action-preview-location] 6)
  require(traad nil t)
  (lambda nil (require 'traad nil t))()
  eval-after-load(python-environment (lambda nil (require 'traad nil t)))
  eval-buffer(#<buffer  *load*-190095> nil "c:/Users/shack/.emacs.d/configuracion.el" nil t)  ; Reading at buffer position 80482
  load-with-code-conversion("c:/Users/shack/.emacs.d/configuracion.el" "c:/Users/shack/.emacs.d/configuracion.el" nil nil)
  load("c:/Users/shack/.emacs.d/configuracion.el" nil nil t)
  load-file("~/.emacs.d/configuracion.el")
  (progn (load-file elfile) (package-initialize) (message "Tiempo de inicialización %.2fs" (float-time (time-subtract (current-time) my-start-time))))
  (let ((orgfile (concat user-emacs-directory "configuracion.org")) (elfile (concat user-emacs-directory "configuracion.el")) (gc-cons-threshold most-positive-fixnum)) (if (or (not (file-exists-p elfile)) (file-newer-than-file-p orgfile elfile)) (progn (my-tangle-config-org orgfile elfile))) (progn (load-file elfile) (package-initialize) (message "Tiempo de inicialización %.2fs" (float-time (time-subtract (current-time) my-start-time)))))
  eval-buffer(#<buffer  *load*> nil "c:/Users/shack/.emacs.d/init.el" nil t)  ; Reading at buffer position 6451
  load-with-code-conversion("c:/Users/shack/.emacs.d/init.el" "c:/Users/shack/.emacs.d/init.el" t t)
  load("c:/Users/shack/.emacs.d/init" t t)
  #f(compiled-function () #<bytecode 0x1000c0c55>)()
  command-line()
  normal-top-level()
```

**Comment by abingham** [comment]
Where is this coming from? There's no mention of `bind-map-set-keys` or `traad--location-button-keymap` in `traad.el`, so it seems like this is the result of your configuration or something.

**Comment by shackra** [comment]
But, I have that in my copy of traad:

```
shack@SANSON-WIN10PRO MSYS ~/.emacs.d
$ grep -rH 'bind-map-set-keys'
elpa/traad-20180720.512/traad.el:(bind-map-set-keys traad--location-button-keymap
Coincidencia en el fichero binario elpa/traad-20180720.512/traad.elc
history:(setq minibuffer-history '("yes" "bind-map-set-keys" "org-download" "issue-2" "mcrc-site" "org-" "org-directory" "jorge" "On (no branch): " "4e99ed2368eeae6de8966907707a155f0b0beaeb" "shackra" "https://github.com/shackra/emacs.git" "github" "python3" "temp" "scrapy" "D:\\shackr\\android-sdk" "10008" "127.0.0.1" "10008" "127.0.0.1" "10008" "127.0.0.1" "10008" "127.0.0.1"))

shack@SANSON-WIN10PRO MSYS ~/.emacs.d
$

```

**Comment by abingham** [comment]
Could you remove that elc and try again?

**Comment by shackra** [comment]
Hello!

Yes, I did that but that did not work.

I would like to point out that those lines actually exists in your source code:

https://github.com/abingham/emacs-traad/blob/1a4bb39dc36ac79fe540923096411e75a6dcfdf4/traad.el#L1494

**Comment by abingham** [comment]
Ah, right. Sorry about that; I'm on vacation right now and somehow was
looking at the wrong version of the code.

I think the problem is that we're missing a dependency on the bind-map
package. Add that and see if it fixes things. If so, could you put together
a PR?

**Comment by shackra** [comment]
For some reason my email responses are not landing here (stupid Windows 10 email client!)

I'll look into this issue and submit a PR.

**Comment by abingham** [comment]
Fixed by #29.
